modules/upgrade/pages/migrate-4x-4x.adoc

```diff
@@ -7,8 +7,8 @@ Migrating {productname} from version 4 to 4.x (when available) can be done usin
 
 [IMPORTANT]
 ====
-Upgrades should be run from a text console, rather than a graphical interface like GNOME.
-If you are logged into a GNOME session running on the machine you are going to update, you will need to switch to a text console.
+Migrations should be run from a text console, rather than a graphical interface like GNOME.
```

Comment:
Now that 4.x is a base product, we don't install WebUI by default, and I wonder if we should even support it.

Reply:
You mean a GUI?

Reply:
I think we cannot prevent users from installing a graphical desktop plus the graphical yast on the server.  As part of the installation documentation we actually mention that the user should select appropriate packages to run the graphical yast.  See https://github.com/SUSE/doc-susemanager/blob/master/modules/installation/pages/install-server-unified.adoc, step 13.

If this is not wanted, we should remove it everywhere.

Reply:
That's right, we can't prevent users from installing a GUI and graphical yast2.

But if we mention it, then users and on their own to find how to get some things, because we don't explain how to do it with console yast2 and GUI yast.

Example: https://github.com/SUSE/doc-susemanager/blob/master/modules/installation/pages/server-setup.adoc

> Log in to the {productname} Server and type yast2 susemanager_setup to begin the setup.
> From the introduction screen select menu:SUSE Manager Setup[Setup SUSE Manager from scratch] and click btn:[Next] to continue.

@moio: what's your take here?

Reply:
Uyuni users can do whatever they prefer, but as far as SUSE Manager is concerned it is supported as an appliance only, meaning no extra packages outside of a handful exceptions.

Running a GUI on top of the SUSE Manager Server is not something we support at all AFAIK.

Reply:
Joe is fine with running a minimal desktop on the server.  Thus closing this for now.